packages/acceptance-tests/pkg-tests-specs/sources/commands/__snapshots__/version.test.js.snap

```diff
@@ -0,0 +1,33 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Commands add it should print the logs to the standard output when using --inline-builds 1`] = `
```

Comment:
This shouldn't be here, right? 🤔

Reply:
good catch ... I generated the initial snapshot with another commands test. I thought I saw the Jest cli say that they were updated though :thinking: